runtime/compiler/codegen/J9RecognizedMethodsEnum.hpp

```diff
@@ -1149,6 +1149,65 @@
    java_lang_invoke_MethodHandleImpl_profileBoolean,
    java_lang_invoke_MethodHandleImpl_isCompileConstant,
 
+   java_lang_invoke_VarHandleBooleans_Array_method,
```

Comment:
Should these be defined only with `J9VM_OPT_OPENJDK_METHODHANDLE`? It looks like they won't be used otherwise anyway

Reply:
That's true, but we have never guarded the other OpenJDK MH-related enum members inside that macro, which is why I left it this way. This is the case throughout the file, where members are not guarded using a macro even if they are only used within a specific build, with the exception of `ENABLE_SPMD_SIMD`. I think there is no real benefit to placing them inside an ifdef, but let me know if you feel otherwise and I will make the change.